### PR TITLE
fix: Take care of clientlaunch.d include on Suse

### PR DESCRIPTION
**Pull request by timdeluxe**
The only real usable Suse package is this: https://software.opensuse.org/download.html?project=home%3Ajochenbecker&package=xymon-client
However it misses to include clientlaunch.d into clientlaunch.cfg, so configured monitors are not picked up.
This PR changes that, by including it for Suse and making it optional for other OS.

### CODE REVIEW

**Comment by timdeluxe** [comment]
#9 should be merged first and then merged into this one to fix the failing kitchen tests...
Please go there first :)

File: data/family/Debian.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+xymon::client::include_clientlaunch_d: false
```
Comment (dploeger) [inline data/family/Debian.yaml]: Why did you add this line as the parameter defaults to false? Just for visibility?

File: data/family/RedHat.yaml [hunk 2]
```diff
@@ -1 +1,2 @@
 xymon::client::config_file: "/etc/sysconfig/xymon-client"
+xymon::client::include_clientlaunch_d: false
```
Comment (dploeger) [inline data/family/RedHat.yaml]: (see Debian.yaml comment)

File: manifests/client.pp [hunk 3]
```diff
@@ -241,6 +246,14 @@
       ensure => 'running'
   }
 
+  if ($include_clientlaunch_d) {
+    file_line { 'include_clientlaunch_d':
+      path => "${xymon_config_dir}/clientlaunch.cfg",
+      line => "directory ${actual_clientlaunch_config}",
+    }
+    File[$actual_clientlaunch_config] -> File_line[include_clientlaunch_d] -> Service[$service_name]
```
Comment (dploeger) [inline manifests/client.pp]: Shouldn't this be ~> Service?

**Review by dploeger — CHANGES_REQUESTED** [review]
Some smaller comments

File: manifests/client.pp [hunk 4]
```diff
@@ -7,6 +7,10 @@
 # @param manage_repository
 #   Manage the repository for package installation
 #
+# @param include_clientlaunch_d
+#   Ensure, that clientlaunch.d/*.cfg is included in clientlaunch.cfg - some xymon packages miss to do so.
+#   Will not remove any include value false!
```
Comment (ThomasMinor) [inline manifests/client.pp]: Could you state this more precisely? What value isn't removed? From where? What is false?
I don't get it.

**Review by ThomasMinor — CHANGES_REQUESTED** [review]

File: data/family/Debian.yaml (hunk 1 above)
Comment (timdeluxe) [reply data/family/Debian.yaml]: well, eh, maybe. You are right, they are obsolete, will remove them

File: data/family/RedHat.yaml (hunk 2 above)
Comment (timdeluxe) [reply data/family/RedHat.yaml]: (see Debian.yaml reply)

File: manifests/client.pp (hunk 3 above)
Comment (timdeluxe) [reply manifests/client.pp]: Hm, yes, makes sense. Will change it.

File: manifests/client.pp (hunk 4 above)
Comment (timdeluxe) [reply manifests/client.pp]: I will to describe it more precisely

File: manifests/client.pp (hunk 3 above)
Comment (timdeluxe) [reply manifests/client.pp]: ...and fixed another error i saw here with missing quotes on the File_line resource...

File: manifests/client.pp (hunk 4 above)
Comment (timdeluxe) [reply manifests/client.pp]: Hope it is more understandable now?

**Review by ThomasMinor — APPROVED** [review]

**Review by dploeger — APPROVED** [review]
Thanks a lot! 🎁
